doc/how_to/interactivity/bind_component.md

```diff
@@ -0,0 +1,51 @@
+# Binding on components
+
+This guide addresses how to bind parameters and bound functions on components.
+
+---
+
+## Binding parameters on components
+
+The power to binding parameters to components can be used with all of Panel's widgets and panes, which makes it easy to create powerful components. An example is by binding a parameter to the `page_size` of the [Tabulator](../../reference/widgets/Tabulator) widget. This will allow you to change the page size of the table with the slider.
+
+```{pyodide}
+import pandas as pd
+import panel as pn
+pn.extension("tabulator")
+
+df = pd.read_csv("https://datasets.holoviz.org/penguins/v1/penguins.csv")
+
+slider = pn.widgets.IntSlider(value=5, start=1, end=10)
+tabulator = pn.widgets.Tabulator(df, page_size=slider, pagination="remote")
+
+pn.Row(slider, tabulator)
+```
+
+# Binding bound function on components
```

Comment:
```suggestion
# Bind functions to multiple widgets
```

This subsection should go into the other guide 'Make your functions interactive', as it is centered on binding functions, not component parameters